samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2195 @@
+{
```

Comment:
<p>In this example, the data used is a univariate timeseries of total monthly rainfall in millimeters. This single variable will be used to forecast the 12 months of rainfall for the months subsequent to the last date in the training data, or put simply, a single variable will be used to predict the future values of that same variable. In the case of a multivariate timeseries model, there would be a list of multiple explanatory variables.</p><p> </p><p>Once the variables are identified, the preprocessing of the data is performed by the <code>prepare_tabulardata</code> method from the <code>arcgis.learn</code> module in the ArcGIS API for Python. This function will take either a non spatial dataframe, a feature layer, or a spatial dataframe containing the dataset as input, and will return a TabularDataObject that can be fed into the model.</p><p> </p><p>The primary input parameters required for the tool are: </p><ul><li>input_features&nbsp;: non spatial dataframe, feature layer, or spatial dataframe containing the primary dataset and the explanatory variables, if there are any</li><li>variable_predict&nbsp;: field name containing the y-variable to be forecasted from the input feature layer/dataframe</li><li>explanatory_variables&nbsp;: list of the field names as 2-sized tuples containing the explanatory variables as mentioned above. Since there are none in this example, it is not required here.</li><li>index_field&nbsp;: field name containing the timestamp</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>